src/mongocrypt-ctx-datakey.c

```diff
@@ -107,7 +107,7 @@ _finalize (mongocrypt_ctx_t *ctx, mongocrypt_binary_t *out)
 {
    _mongocrypt_ctx_datakey_t *dkctx;
    bson_t key_doc, child;
-   int64_t current_time_ms;
+   struct timeval tp;
```

Comment:
This fix is in master per https://github.com/mongodb/libmongocrypt/commit/f9bde7222499314e561fabe8f21587ddbdbb169f

I'm not sure why it's showing up in this PR.

Reply:
this needs to be rebased off of #39. H/o, I will update them.